doc/radosgw/notifications.rst

```diff
@@ -138,7 +138,7 @@ Request parameters:
 
  - URI: ``amqp://[<user>:<password>@]<fqdn>[:<port>][/<vhost>]``
  - user/password defaults to: guest/guest
- - user/password may only be provided over HTTPS. Topic creation request will be rejected if not
+ - user/password may only be provided over HTTPS. If not,topic creation request will be rejected.
```

Comment:
rephrasing is ok, but I think there is a space missing after the comma?

Reply:
Thanks I have checked spacing for all , other then two mentioned.